react/features/toolbox/components/web/Toolbox.js

```diff
@@ -553,6 +560,21 @@ class Toolbox extends Component<Props, State> {
 
         this._doToggleChat();
     }
+	
+	_onShortcutToggleVideoQuality: () => void;
+
+    /**
+     * Creates an analytics keyboard shortcut event and dispatches an action for
+     * toggling the display of Video Quality.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onShortcutToggleVideoQuality() {
+        sendAnalytics(createShortcutEvent('Video.quality'));
+
+        this._doOpenVideoQuality();
```

Comment:
Does this method open or toggle (open or close) the dialog? If we're going to have a keyboard shortcut maybe it should be a toggle.